src/resources/js/test/usersTab.test.tsx

```diff
@@ -319,7 +319,7 @@ describe('check the contents of all the fields displayed in the editor', () => {
     const userType = await screen.findByRole('button', { name: 'User Is A:' });
     // the replace method below gets rid of zero-width spaces
     // TODO: is there a better way to do this?  Maybe get tabindex, or some other way to find the current selection?
-    expect(userType.textContent?.replace(/\u200B/g, '')).toBe('');
+    expect(userType.textContent?.replace(/\u200B/g, '')).toBe('Service Body Administrator');
```

Comment:
change to 
expect(userType.textContent).toBe('Service Body Administrator');

the replace thing was a hack, needed only when the content was just whitespace

Reply:
fixed. I was looking at that lol, but figured this is what you had, so ill just roll with it. Passed the test tho.